akka-stream/src/main/scala/akka/stream/impl/fusing/StreamOfStreams.scala

```diff
@@ -503,6 +507,7 @@ final class Split[T](val decision: Split.SplitDecision, val p: T ⇒ Boolean, va
       }
 
       override def onPull(): Unit = {
+        cancelTimer(SubscriptionTimer)
```

Comment:
A map lookup on each pull after the first, but I guess it doesn't matter performance wise as the map will be empty?